libobs/obs.c

```diff
@@ -1027,6 +1023,14 @@ void obs_shutdown(void)
 	stop_video();
 	stop_hotkeys();
 
+	module = obs->first_module;
+	while (module) {
+		struct obs_module *next = module->next;
+		free_module(module);
+		module = next;
+	}
+	obs->first_module = NULL;
+
```

Comment:
This seems to go beyond "Remove excessive null checks", should probably be in another commit (e.g. the behavior change seems to be too big to be buried in the last paragraph of a commit message with a different title)

Reply:
I've split this into two separate commits now.